tests/baselines/reference/classdecl.js

```diff
@@ -220,11 +220,10 @@ declare class a {
         x: number;
         y: number;
     };
-    private static d2();
+    private static d2;

```

Comment:
I don't imagine the parameter lists were ever necessary to meet the purpose of emitting these private members anyways (to avoid collisions), unless there is a use case I'm not aware of that would make this a breaking change.

Reply:
We have an open issue to make this change, too, as I said in the OP.

Reply:
My statement was meant to indicate 👍 to this change, though I'm just unsure if there are any valid reasons to keep it as it was.